client/client-multi/client-cpp/Modules/FindSqlite3.cmake

```diff
@@ -37,31 +29,8 @@ IF (APPLE)
   ENDIF ()
 ENDIF (APPLE)
 
-IF ($ENV{SQLITE_ROOT})
```

Comment:
Such changes will affect the process of building Kaa SDK, thus need to be discussed.